src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
+  const menuItems = [
+    {
+      html: "Logout",
```

Comment:
Should we keep the `Sign out` we had before this change?

Reply:
What do you mean exactly? We are removing the sign out from the sidebar.

Reply:
I mean the menu item, had a label called `Sign out` that we changed to the word `Logout`. I believe we should still call it `Sign out`.

Reply:
@leemunroe what are your thoughts on this?

Reply:
Ah lets keep `Sign Out`. Oversight on my part. Not deliberate.